### PR TITLE
renamed haushaltsgeräte to home appliance

### PR DESCRIPTION
**Pull request by eckelj**
* rename Haushaltsgeraete to home appliance

### CODE REVIEW

**Comment by NormannK** [comment]
I'd love to see the whole class file renamed too. What do you think? 
This Pr gets us into reach of the first milestone. Thank you very much

**Comment by eckelj** [comment]
@NormannK I am in for that, but I didn't want to do it right now due to potential conflicts with others. By the way, do we need to contain the name class in the filename? What do you suggest?

**Comment by NormannK** [comment]
I have not enough experience to give a good advice with the naming. I would use the class name in the file name to make it obvious and not to twist brains.

**Comment by NormannK** [comment]
Also why did I had to approve the test runs here?

**Comment by eckelj** [comment]
@NormannK I added the suggested changes. but I don#t know why you have to approve the test execution. is it because I PR from my fork?

**Comment by NormannK** [comment]
Should not be. Often I do the same

**Comment by SREbuilt** [comment]
on the recent file name rename, I noticed a small typo: an 'e' is missing:

- instead of class_home_applianc.py
- it should be class_home_appliance.py

(@eckelj
[renamed file to class_home_applianc.py](https://github.com/Akkudoktor-EOS/EOS/pull/196/commits/5739440d7134797c286491a67e2d0316c64ca8dc))

File: src/akkudoktoreos/visualize.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def visualisiere_ergebnisse(
         )
         plt.plot(
             hours,
-            ergebnisse["Haushaltsgeraet_wh_pro_stunde"],
+            ergebnisse["Home_appliance_wh_per_hour"],
```
Comment (SREbuilt) [inline src/akkudoktoreos/visualize.py:122]: can this be refactored to results[...]?

File: README.md [hunk 2]
```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
...
+    - **Length**: 35
+  - **Einnahmen_Euro_pro_Stunde**: An array of floats representing the simulated income in euros per hour.
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Gesamt_Verluste**: The total simulated losses in watt-hours.
+    - **Type**: Float
+  - **Gesamtbilanz_Euro**: The total simulated balance in euros.
+    - **Type**: Float
+  - **Gesamteinnahmen_Euro**: The total simulated income in euros.
```
Comment (SREbuilt) [inline README.md:318]: simulated_gain_EUR
or
simulated_profit_EUR

File: README.md [hunk 3]
```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
...
+    - **Type**: Float
+  - **Gesamtkosten_Euro**: The total simulated costs in euros.
```
Comment (SREbuilt) [inline README.md:320]: simulated_total_cost_EUR

File: README.md [hunk 4]
```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
...
+    - **Type**: Float
+  - **Home_appliance_wh_per_hour**: An array of floats representing the simulated energy consumption of a household appliance in watt-hours per hour.
```
Comment (SREbuilt) [inline README.md:322]: simulated_consumption_home_appliances_Wh_per_hour

(bits are cheap, these days ;))

File: README.md [hunk 5]
```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
...
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Kosten_Euro_pro_Stunde**: An array of floats representing the simulated costs in euros per hour.
```
Comment (SREbuilt) [inline README.md:326]: simulated_cost_EUR_per_hour

File: README.md [hunk 6]
```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
...
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Netzbezug_Wh_pro_Stunde**: An array of floats representing the simulated grid consumption in watt-hours per hour.
```
Comment (SREbuilt) [inline README.md:330]: simulated_energy_import_Wh_per_hour

File: README.md [hunk 7]
```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
...
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Netzeinspeisung_Wh_pro_Stunde**: An array of floats representing the simulated grid feed-in in watt-hours per hour.
```
Comment (SREbuilt) [inline README.md:334]: simulated_energy_export_Wh_per_hour

File: README.md [hunk 8]
```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
...
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Verluste_Pro_Stunde**: An array of floats representing the simulated losses per hour.
```
Comment (SREbuilt) [inline README.md:338]: simulated_losses_EUR_per_hour
or
simulated_financial_losses_EUR_per_hour

File: README.md [hunk 9]
```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
...
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **akku_soc_pro_stunde**: An array of floats representing the simulated state of charge of the battery in percentage per hour.
```
Comment (SREbuilt) [inline README.md:342]: simulated_PV_battery_SoC_per_hour
or
simulated_PV_battery_pct_per_hour

**Review by SREbuilt — COMMENTED** [review]
prosing consistent naming conventions... and having everything in English
and longer names provide more clarity.. (bits are cheap, these days 🤗)

File: src/akkudoktoreos/visualize.py (hunk 1 above)
Comment (NormannK) [reply src/akkudoktoreos/visualize.py:115]: will be done on an other PR

**Review by NormannK — APPROVED** [review]

File: README.md (hunk 8 above)
Comment (NormannK) [reply README.md]: was done reworked in #204

File: README.md (hunk 7 above)
Comment (NormannK) [reply README.md]: was done reworked in #204

File: README.md (hunk 6 above)
Comment (NormannK) [reply README.md]: was done reworked in #204

File: README.md (hunk 5 above)
Comment (NormannK) [reply README.md]: was done reworked in #204

File: README.md [hunk 10]
```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
...
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Einnahmen_Euro_pro_Stunde**: An array of floats representing the simulated income in euros per hour.
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Gesamt_Verluste**: The total simulated losses in watt-hours.
+    - **Type**: Float
+  - **Gesamtbilanz_Euro**: The total simulated balance in euros.
```
[the remark this answers is not in the document]
Comment (NormannK) [reply README.md]: was done reworked in #204

File: README.md [hunk 11]
```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
...
-  - **Length**: 35
+  - **E-Auto_SoC_pro_Stunde**: An array of floats representing the simulated state of charge of the electric car per hour.
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Eigenverbrauch_Wh_pro_Stunde**: An array of floats representing the simulated self-consumption in watt-hours per hour.
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Einnahmen_Euro_pro_Stunde**: An array of floats representing the simulated income in euros per hour.
```
[the remark this answers is not in the document]
Comment (NormannK) [reply README.md]: done. reworked in #204

File: README.md [hunk 12]
```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
...
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Gesamt_Verluste**: The total simulated losses in watt-hours.
```
[the remark this answers is not in the document]
Comment (NormannK) [reply README.md]: done. reworked in #204

File: README.md [hunk 13]
```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
...
-  - **Length**: 35
-- **akku_soc_pro_stunde**: An array of floats representing the simulated state of charge of the battery in percentage per hour.
-  - **Type**: Array
-  - **Element Type**: Float
-  - **Length**: 35
+  - **E-Auto_SoC_pro_Stunde**: An array of floats representing the simulated state of charge of the electric car per hour.
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Eigenverbrauch_Wh_pro_Stunde**: An array of floats representing the simulated self-consumption in watt-hours per hour.
```
[the remark this answers is not in the document]
Comment (NormannK) [reply README.md]: done. reworked in #204

File: README.md [hunk 14]
```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
...
-  - **Length**: 35
-- **Verluste_Pro_Stunde**: An array of floats representing the simulated losses per hour.
-  - **Type**: Array
-  - **Element Type**: Float
-  - **Length**: 35
-- **akku_soc_pro_stunde**: An array of floats representing the simulated state of charge of the battery in percentage per hour.
-  - **Type**: Array
-  - **Element Type**: Float
-  - **Length**: 35
+  - **E-Auto_SoC_pro_Stunde**: An array of floats representing the simulated state of charge of the electric car per hour.
```
[the remark this answers is not in the document]
Comment (NormannK) [reply README.md]: done. reworked in #204

File: README.md (hunk 2 above)
Comment (NormannK) [reply README.md]: done. reworked in #204

File: README.md (hunk 3 above)
Comment (NormannK) [reply README.md]: done. reworked in #204

File: README.md (hunk 4 above)
Comment (NormannK) [reply README.md]: done. reworked in #204

**Pull request merged**